### PR TITLE
FEAT(#69): add home page with content

### PR DESCRIPTION
**Pull request by Darkosz1012**

### CODE REVIEW

**Review by frogi16 — CHANGES_REQUESTED** [review]
Looks OK, maybe it's to big? Empty space on the sides could look better

File: front/src/components/HomePage/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    faMagic,
+    faThumbsUp,
+} from "@fortawesome/free-solid-svg-icons";
+
+function HomePage() {
+    return (
+        <div className="HomePage">
+            <header className="HomePage__header">
+                <h1>
+                    Witajcie na stronie orgazniatora zawodów łuczniczych
```
Comment (frogi16) [inline front/src/components/HomePage/HomePage.js:17]: Typo
```suggestion
                    Witajcie na stronie organizatora zawodów łuczniczych
```

File: front/src/components/HomePage/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                    Amazonka!
+                </h1>
+            </header>
+            <div className="container">
+                <h2>Czym jest Amazonka?</h2>
+                <p>
+                    Jest to odpowiedź na brak komunikacji między zawodnikiem a
+                    organizatorem oraz słabą automatyzacji zawodów łuczniczych w
```
Comment (frogi16) [inline front/src/components/HomePage/HomePage.js:25]: Typo: automatyzację

File: front/src/components/HomePage/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                    naszym kraju. Dzięki Amazonce dostajesz możliwość
+                    komunikowania zawodnikom podstawowych informacji o zawodach
+                    takich jak harmonogram, rozstawienie oraz wyniki bez żadnych
+                    pośredników i natychmiast po jednym kliknięciu. Dodatkowo
+                    zapewniamy automatyzację w podstawowych czynnościach
+                    organizacyjnych, które zawsze wyglądają tak samo np.
+                    sumowanie wyników rozpisywanie eliminacji.
```
Comment (frogi16) [inline front/src/components/HomePage/HomePage.js:32]:
```suggestion
                    sumowanie wyników, czy rozpisywanie eliminacji.
```

File: front/src/components/HomePage/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+                        <FontAwesomeIcon
+                            className="HomePage__card-icon"
+                            icon={faCogs}
+                        />
+                        <h3 className="HomePage__card-heading">
+                            Automatyczne podliczanie wyników
+                        </h3>
+                        <p className="HomePage__card-content">
+                            Na naszej witrynie możesz wpisywać pojedyncze
+                            strzały lub wynik serii, a nasz system wyliczy
```
Comment (frogi16) [inline front/src/components/HomePage/HomePage.js:46]: Repetition
```suggestion
                            strzały lub wynik serii, a system wyliczy
```

File: front/src/components/HomePage/HomePage.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+                            className="HomePage__card-icon"
+                            icon={faHistory}
+                        />
+                        <h3 className="HomePage__card-heading">
+                            Wyniki dostępne dla użytkowników na żywo
+                        </h3>
+                        <p className="HomePage__card-content">
+                            Po wpisaniu każdej serii w naszym systemie, gdy
+                            klikniesz "podlicz wyniki", udostępniasz najnowszy
+                            ranking wszystkim osobą, które wejdą na tą stronę.
```
Comment (frogi16) [inline front/src/components/HomePage/HomePage.js:76]: Spelling errors
```suggestion
                            ranking wszystkim osobom, które wejdą na tę stronę.
```

File: front/src/components/HomePage/HomePage.js [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+                        <FontAwesomeIcon
+                            className="HomePage__card-icon"
+                            icon={faListOl}
+                        />
+                        <h3 className="HomePage__card-heading">
+                            Automatyczne generowanie rozstawienia
+                        </h3>
+                        <p className="HomePage__card-content">
+                            Wystarczy, że wpiszesz zawodników do naszego systemu
+                            i przydzielisz im kategorie, a nasz system
```
Comment (frogi16) [inline front/src/components/HomePage/HomePage.js:89]: Repetition
```suggestion
                            i przydzielisz im kategorie, a system
```

File: front/src/components/HomePage/HomePage.js [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+                    <div class="HomePage__card">
+                        <FontAwesomeIcon
+                            className="HomePage__card-icon"
+                            icon={faMagic}
+                        />
+                        <h3 className="HomePage__card-heading">
+                            Automatyczne przetwarzanie eliminacji
+                        </h3>
+                        <p className="HomePage__card-content">
+                            Na podstawie wyników kwalifikacji, nasz system
```
Comment (frogi16) [inline front/src/components/HomePage/HomePage.js:102]: Repetition, needless comma
```suggestion
                            Na podstawie wyników kwalifikacji system
```

**Review by frogi16 — CHANGES_REQUESTED** [review]

File: front/src/components/HomePage/HomePage.js (hunk 2 above)
Comment (Darkosz1012) [inline front/src/components/HomePage/HomePage.js:25]:
```suggestion
                    organizatorem oraz słabą automatyzację zawodów łuczniczych w
```

**Review by frogi16 — CHANGES_REQUESTED** [review]
I am not sure if removing package-lock is a good idea, especially without adding it to gitignore

**Comment by Darkosz1012** [comment]
> I am not sure if removing package-lock is a good idea, especially without adding it to gitignore

I didn't remove packege-lock. It rebuild itself, when I use "npm i", so I think there should not be a problem.

**Review by frogi16 — APPROVED** [review]

**Review by pieczonkam — APPROVED** [review]

**Pull request merged**